sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClient.java

```diff
@@ -145,7 +144,7 @@
                 recognizeCustomFormsOptions.getPollInterval(),
                 urlActivationOperation(() ->
                     service.analyzeWithCustomModelWithResponseAsync(UUID.fromString(modelId),
-                        isFieldElementsIncluded, new SourcePath().setSource(formUrl), context)
+                        isFieldElementsIncluded, null, new SourcePath().setSource(formUrl), context)
```

Comment:
is the null because of the new `pages` parameter added?

Reply:
yes!